src/app/modules/project/details/bottom-drawer/bottom-drawer.component.html

```diff
@@ -7,6 +7,7 @@
     </div>
     <app-settings-dropdown
         (projectHighlighted)="isProjectHighlighted = $event"
+        (editButtonClicked)='this.editMode.emit($event)'
```

Comment:
It's up to you, but personally I would make a function for these, makes your code cleaner, and debugging will be easier too. We should probably include something for this in the code guidelines as well